ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerServices/LocalServerServices.cpp

```diff
@@ -78,8 +78,13 @@ void LocalServerServices::connectSignals()
       }
       case LocalServerStatus::Started:
       {
-        // set temp path
-        QTemporaryDir tempDir;
+        // create temp path
+        const QString tempPath = LocalServerServices::shortestTempPath() + "/EsriQtTemp";
+
+        // create the directory
+        const QTemporaryDir tempDir(tempPath);
```

Comment:
My bad for not noticing this before, but QTemporaryDir  needs to be a class member for this to work (probably a `std::unique_ptr<QTemporaryDir>`). This class deletes the temp dir on destruction, and it will destruct at the end of this block.